MANIFEST.in

```diff
@@ -1,4 +1,4 @@
 include LICENSE NEWS zonefile_metadata.json updatezinfo.py pyproject.toml
-recursive-include dateutil/test *
+recursive-include src/dateutil/test *
```

Comment:
I think we actually want to move `dateutil/test` to `tests/` as a sibling directory of `src/`.

Reply:
I'd like to do that in a separate commit/PR.

Reply:
I suppose. It feels like moving the tests out of the package is a significant part of the advantage of using the src/ layout because:

1. It makes it so we no longer need to exclude `tests/` from installation
2. I'm not entirely confident that we can "test as installed" when running the (not installed) tests from inside the package, because `pytest` messes around with the `PYTHONPATH` / `sys.path`: https://docs.pytest.org/en/6.2.x/pythonpath.html

It seems prudent to take these two changes as a single bundle, since traditionally the "src layout" means "src sibling to tests", at least in my mind.